bookmarks.html

```diff
@@ -10,29 +10,26 @@
     <header>
       <h1>Quiz App</h1>
     </header>
-    <!-- I will use question-card as a class later to style it - What is better to use - or _ when I write a class?  -->
     <section class="question-card">
-      <div class="div_question-card">
+      <div class="question-card__button">
         <h2>What is Riesling?</h2>
         <button aria-label="Klick here to get answere">Show Answer</button>
         <p>A Grape variety, that was esay</p>
         <button>Hide Answer</button>
       </div>
-      <!-- I will use categories as a class to style it later -->
-      <ul class="question-card_categories">
-        <li class="question-card_categories_list">#wine</li>
-        <li class="question-card_categories_list">#basic</li>
-        <li class="question-card_categories_list">#beverage</li>
+      <ul class="question-card__categories">
+        <li class="question-card__categories--list">#wine</li>
+        <li class="question-card__categories--list">#basic</li>
+        <li class="question-card__categories--list">#beverage</li>
       </ul>
-      <!-- I have to add a Element for a bookmark as well, simple as a <P> Element? -->
       <p><img class="wine-bottle" src="/icons/wine-bottle.svg"></img></p>
     </section>
     <footer>
       <nav>
         <ul class="nav">
-          <li><a class="nav_link" href="index.html">Quiz</a></li>
-          <li><a class="nav_link" href="bookmarks.html">Bookmarks</a></li>
-          <li><a class="nav_link" href="profile.html">Profile Page</a></li>
+          <li><a class="nav__list-item" href="index.html">Quiz</a></li>
+          <li><a class="nav__list-item" href="bookmarks.html">Bookmarks</a></li>
+          <li><a class="nav__list-item" href="profile.html">Profile Page</a></li>
```

Comment:
When we use `nav__list` on our `ul`, then this also makes much more sense, and it's a good solution.

Reply:
Got it now, thanks for that.